packages/components/src/components/accordion/_accordion.scss

```diff
@@ -175,6 +184,54 @@
   }
 }
 
+@function px($value) {
+  @if unit($value) == 'px' {
+    @return $value;
+  }
+  @if unit($value) == 'rem' {
+    @return ($value / ($value * 0 + 1)) * 16px;
+  }
+  @error "Unable to convert unit #{unit($value)} to px";
+}
+
+@function rem($value) {
+  @if unit($value) == 'rem' {
+    @return $value;
+  }
+  @if unit($value) == 'px' {
+    @return ($value / 16px) * 1rem;
+  }
+  @error "Unable to convert unit #{unit($value)} to px";
+}
+
+@function get-theme-value($token, $field) {
+  @return map-get(map-get($carbon--theme, $token), $field);
```

Comment:
what does map-get return or look like? I _think_ I understand the overall use of this (we're getting the line height for a certain token), but I've never seen a nested map-get so I don't quite get what this return looks like.